src/opcode.def

```diff
@@ -282,6 +282,7 @@ WABT_OPCODE(V128, V128, V128, ___, 0, 0xfd, 0x3b,  V128And, "v128.and")
 WABT_OPCODE(V128, V128, V128, ___, 0, 0xfd, 0x3c,  V128Or,  "v128.or")
 WABT_OPCODE(V128, V128, V128, ___, 0, 0xfd, 0x3d,  V128Xor, "v128.xor")
 WABT_OPCODE(V128, V128, ___,  ___, 0, 0xfd, 0x3e,  V128Not, "v128.not")
+WABT_OPCODE(V128, ___,  ___,  ___, 0, 0xfd, 0x3f,  V128BitSelect, "v128.bitselect")
```

Comment:
This can be specified with 4 `V128`s as well, I believe.

Reply:
Done.